src/internal/storage/fileset/fileset.go

```diff
@@ -32,6 +32,11 @@ func (id ID) HexString() string {
 	return string(id)
 }
 
+// TrackerID returns the ID of the fileset's tracker object.
+func (id ID) TrackerID() string {
+	return filesetObjectID(id)
```

Comment:
Lets go ahead and get rid of this helper function and implement it here. Then, replace the places with the calls to it with `TrackerID`.